data/apply-planet_osm_point.sql

```diff
@@ -6,6 +6,14 @@ BEGIN
 --------------------------------------------------------------------------------
 
 CREATE INDEX planet_osm_point_place_index ON planet_osm_point(place) WHERE name IS NOT NULL AND place IN ('borough', 'city', 'continent', 'country', 'county', 'district', 'farm', 'hamlet', 'island', 'isolated_dwelling', 'lake', 'locality', 'neighbourhood', 'ocean', 'province', 'quarter', 'sea', 'state', 'suburb', 'town', 'village');
-CREATE INDEX planet_osm_point_level_index ON planet_osm_point(mz_calculate_poi_level("aerialway", "aeroway", "amenity", "barrier", "craft", "highway", "historic", "leisure", "lock", "man_made", "natural", "office", "power", "railway", "shop", "tourism", "waterway")) WHERE mz_calculate_poi_level("aerialway", "aeroway", "amenity", "barrier", "craft", "highway", "historic", "leisure", "lock", "man_made", "natural", "office", "power", "railway", "shop", "tourism", "waterway") IS NOT NULL;
+CREATE INDEX planet_osm_point_level_index ON planet_osm_point(mz_calculate_poi_level("aerialway", "aeroway", "amenity", "barrier", "craft", "highway", "historic", "leisure", "lock", "man_made", "natural", "office", "power", "railway", "shop", "tourism", "waterway", 0::real)) WHERE mz_calculate_poi_level("aerialway", "aeroway", "amenity", "barrier", "craft", "highway", "historic", "leisure", "lock", "man_made", "natural", "office", "power", "railway", "shop", "tourism", "waterway", 0::real) IS NOT NULL;
```

Comment:
As I understand it, Postgres can only [combine index scans](http://www.postgresql.org/docs/9.3/static/indexes-bitmap-scans.html) under some pretty specific circumstances and I haven't seen it do so on my database. I think the generally-accepted method would be to create something like the following:

```SQL
CREATE INDEX planet_osm_point_poi_zoom_9 ON planet_osm_point USING gist(way) WHERE mz_poi_min_zoom <= 9;
CREATE INDEX planet_osm_point_poi_zoom_10 ON planet_osm_point USING gist(way) WHERE mz_poi_min_zoom <= 10;
CREATE INDEX planet_osm_point_poi_zoom_11 ON planet_osm_point USING gist(way) WHERE mz_poi_min_zoom <= 11;
CREATE INDEX planet_osm_point_poi_zoom_12 ON planet_osm_point USING gist(way) WHERE mz_poi_min_zoom <= 12;
CREATE INDEX planet_osm_point_poi_zoom_13 ON planet_osm_point USING gist(way) WHERE mz_poi_min_zoom <= 13;
```

Each index is trading off some improved runtime performance for slower updates, so it's probably not worth doing every single zoom, perhaps only 9, 11 & 13.